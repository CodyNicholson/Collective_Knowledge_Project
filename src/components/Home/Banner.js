import React from 'react';

const Banner = ({ appName, token, declaration }) => {
  if (token) {
    return null;
  }
  return (
    <div className="banner">
      <div className="container">
        <h1 className="logo-font">
          {appName}
        </h1>
        <p>{declaration}</p>
      </div>
    </div>
  );
};

export default Banner;
