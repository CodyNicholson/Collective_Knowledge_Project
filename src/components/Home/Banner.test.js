import React from 'react';
import Banner from './Banner';
import renderer from 'react-test-renderer';

test('Banner page appears', () => {
  const component = renderer.create(
  	<div>
    	<Banner></Banner>
    	<Banner token={false} appName="Collective Knowledge" declaration="What we're all about"/>
    	<Banner token={true} appName="Collective Knowledge" declaration="What we're all about"/>
  	</div>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});