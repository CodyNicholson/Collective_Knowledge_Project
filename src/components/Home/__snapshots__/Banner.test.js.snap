// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner page appears 1`] = `
<div>
  <div
    className="banner"
  >
    <div
      className="container"
    >
      <h1
        className="logo-font"
      />
      <p />
    </div>
  </div>
  <div
    className="banner"
  >
    <div
      className="container"
    >
      <h1
        className="logo-font"
      >
        Collective Knowledge
      </h1>
      <p>
        What we're all about
      </p>
    </div>
  </div>
</div>
`;
